sql/core/src/test/scala/org/apache/spark/sql/execution/SortSuite.scala

```diff
@@ -18,9 +18,13 @@
 package org.apache.spark.sql.execution
 
 import org.apache.spark.sql.catalyst.expressions.{BoundReference, Ascending, SortOrder}
+import org.apache.spark.sql.catalyst.dsl.expressions._
+
+import org.apache.spark.sql.test.TestSQLContext
 import org.apache.spark.sql.types.{IntegerType, StringType}
 
 class SortSuite extends SparkPlanTest {
+  import TestSQLContext.implicits.localSeqToDataFrameHolder
```

Comment:
Actually, what's the right way to create such an implicit alias?  I don't remember the trick offhand.